pkg/cmd/cli/cmd/tag.go

```diff
@@ -267,7 +267,7 @@ func (o TagOptions) Validate() error {
 	}
 
 	if o.deleteTag && o.aliasTag {
-		return errors.New("--alias and --delete may not both be specified")
+		return errors.New("--alias and --delete should not be used together")
```

Comment:
Yeah, go with the old form, the only thing you can fix is `may not be both specified`